<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="border_function">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>1011</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="heart_rate">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>180</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>HEART RATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fall_alarm">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>500</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>FALL ALARM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="breathing_rate">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>BREATHING RATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sleep_phase">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>SLEEP PHASE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="activities">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>340</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>ACTIVITIES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sleep_position">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;SLEEP POSITION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);
</string>
   </property>
   <property name="text">
    <string>SLEEP POSITION</string>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>70</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;CHOOSE FUNCTION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_out">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>0</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgba(0, 0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_account_for_nurse">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>660</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CREATE ACCOUNT &lt;/p&gt;&lt;p&gt;FOR PATIENT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>CREATE ACCOUNT 
 FOR NURSE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
